Sources/Amplitude/Amplitude.m

```diff
@@ -87,6 +87,8 @@ @interface Amplitude()
 @property (nonatomic, assign) long long sessionId;
 @property (nonatomic, assign) BOOL backoffUpload;
 @property (nonatomic, assign) int backoffUploadBatchSize;
+@property (nonatomic, copy, readwrite, nullable) NSString *userId;
```

Comment:
This will allow user to call setter without changing `readonly` for `userId` and `deviceId` inside `Amplitude.h`?

Reply:
Changed so readwrite occurs in header instead of implementation

Reply:
Will revert so that we can solve the method syntax bug